package.json

```diff
@@ -46,4 +54,4 @@
     "tslint": "^6.1.2",
     "tslint-config-prettier": "^1.18.0"
   }
-}
+}
```

Comment:
NIT: No ending newline

Reply:
done

Reply:
I tested `glob.sync(${cwd}/performance/*);` vs. `glob.sync(${cwd}/performance/**/*);` as well. first one included only files in performance. The second one includes the files in performance and in the subfolder in performance. I think that we should go with `x/**/*`